src/vs/workbench/parts/terminal/electron-browser/terminalInstance.ts

```diff
@@ -52,7 +55,7 @@ export class TerminalInstance {
 				this.isExiting = true;
 				this.dispose();
 				if (exitCode) {
-					console.error('Integrated terminal exited with code ' + exitCode);
+					this.messageService.show(Severity.Error, nls.localize('terminal.integrated.exitedWithCode', 'Terminal: The terminal process terminated with exit code: {0}', exitCode));
```

Comment:
I don't think `Terminal: ` is necessary here, doesn't seem to be used elsewhere (from a cursory search of `Severity.Error`).